app.test.js

```diff
@@ -0,0 +1,81 @@
+const app = require("./app.js");
+const fs = require("fs/promises");
+const request = require("supertest");
+const seed = require("./db/seeds/seed.js");
+const connection = require("./db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("./db/data/test-data/index.js");
+const express = require("express");
+const { parse } = require("path");
+app.use(express.json());
+
+afterAll(() => {
+  connection.end();
```

Comment:
as this is on a new line there is no implicit return, this may need a return adding before connection.end()